pages/instrumenting.md

```diff
@@ -114,10 +120,10 @@ order to reassemble a complete trace. Five pieces of information are required:
 * Trace Id
 * Span Id
 * Parent Id
-* Is Sampled
```

Comment:
good point.

ps if this document doesn't already, it should mention the standard http form of these properties: https://github.com/openzipkin/brave/blob/e474ed1e1cd291c7ebc6830c58fdba0a6318fdd2/brave-http/src/main/java/com/github/kristofa/brave/http/BraveHttpHeaders.java